keylime/ima/ima_dm.py

```diff
@@ -12,14 +12,19 @@
 from typing import Dict, List, Optional, Union
 
 import lark
-from lark import Lark, Transformer, v_args
 from packaging import version
 
 from keylime.common.algorithms import Hash
 from keylime.failure import Component, Failure
 from keylime.ima import ast
 from keylime.ima.dm_grammar import DM_GRAMMAR
 
+if sys.version_info >= (3, 10):
```

Comment:
Maybe add a comment why this is needed for Python 3.10?

Reply:
Yes, that is funny. Does lark namespace for a fixed version changes based on the Python version? I never saw that : )

Reply:
yes, at least that is what I inferred looking at the issues reported by the type checker and the code of the Lark module itself at different versions. Of course I might be wrong, feel free to double-check. I added a minimal comment for now: d84fa2c

Reply:
Can you investigate with which lark versions this happens?

Reply:
My setup is Python 3.10.6 and Lark 1.1.3. If i Try the "old-style" import I get this for `Lark`:

```console
"Lark" is not exported from module "lark"
  Import from "lark.lark" instead
```

(same thing for `Transformer` and `v_args`)

Reply:
Can you check if Lark 0.X versions and other Python versions, because making it conditionally on the Python version still does not seem to be right.

Reply:
Apparently it is a Lark issue and they are working on it: https://github.com/lark-parser/lark/issues/1199. To my understanding, the "old-style" import is intended to be supported, so I would simply roll back to it.

Reply:
done, hopefully the type checker will eventually be happy once they fix.